tests/test_serialize.py

```diff
@@ -296,8 +296,7 @@ def test_mixed_tensor_return(self, monkeypatch, ObsFunc):
         "ObsStructC128" and "ObsStructC64" not in dir(pennylane_lightning.lightning_qubit_ops),
         reason="ObsStructC128 and ObsStructC64 are required",
     )
-    @pytest.mark.parametrize("ObsFunc", [ObsStructC128, ObsStructC64])
```

Comment:
Are these separated out now into different tests?

Reply:
Yes. There are test cases (`test_integration_c64` and `test_integration_c128`) for each one. I had to remove this to prevent `ComplexWarning` due to some casting complex values to real issues.